sycl/test-e2e/MemorySanitizer/lit.local.cfg

```diff
@@ -0,0 +1,11 @@
+config.substitutions.append(
+    ("%device_msan_flags", "-Xarch_device -fsanitize=memory")
+)
+config.substitutions.append(
+    ("%force_device_msan_rt", "env UR_ENABLE_LAYERS=UR_LAYER_MSAN")
+)
+
+config.unsupported_features += ['cuda', 'hip']
```

Comment:
Is this not covered by the `REQUIRES` statement in the tests requiring `level_zero` for GPUs?

Reply:
Oh, you're right, they are redundant.